pkg/service/v0/option.go

```diff
@@ -35,17 +34,26 @@ func Logger(val log.Logger) Option {
 	}
 }
 
-// Config provides a function to set the config option.
-func Config(val *config.Config) Option {
+func Database(val *config.Config) Option {
+	return func(o *Options) {
+		o.Config = val
+	}
+}
+
+func Table(val *config.Config) Option {
 	return func(o *Options) {
 		o.Config = val
 	}
 }
 
-// Middleware provides a function to set the middleware option.
-func Middleware(val ...func(http.Handler) http.Handler) Option {
+func Nodes(val *config.Config) Option {
 	return func(o *Options) {
-		o.Middleware = val
+		o.Config = val
 	}
 }
 
+func Config(val *config.Config) Option {
```

Comment:
Codacy found an issue: [exported function Config should have comment or be unexported](https://app.codacy.com/gh/owncloud/ocis-store/pullRequest?prid=5855622)